expected/wasm32-wasi/predefined-macros.txt

```diff
@@ -2290,53 +2324,53 @@
 #define _SC_XOPEN_XPG2 98
 #define _SC_XOPEN_XPG3 99
 #define _SC_XOPEN_XPG4 100
-#define _SEARCH_H 
-#define _SEMAPHORE_H 
-#define _SIZE_T 
-#define _STDALIGN_H 
-#define _STDBOOL_H 
-#define _STDC_PREDEF_H 
-#define _STDINT_H 
-#define _STDIO_EXT_H 
-#define _STDIO_H 
-#define _STDLIB_H 
-#define _STDNORETURN_H 
-#define _STRINGS_H 
-#define _STRING_H 
-#define _STROPTS_H 
-#define _SYSEXITS_H 
-#define _SYS_EVENTFD_H 
-#define _SYS_FILE_H 
-#define _SYS_IOCTL_H 
-#define _SYS_PARAM_H 
-#define _SYS_RANDOM_H 
-#define _SYS_REG_H 
-#define _SYS_SELECT_H 
-#define _SYS_SOCKET_H 
-#define _SYS_STAT_H 
-#define _SYS_SYSCALL_H 
-#define _SYS_SYSINFO_H 
-#define _SYS_TIMEB_H 
-#define _SYS_TIMEX_H 
-#define _SYS_TIME_H 
-#define _SYS_TTYDEFAULTS_H 
-#define _SYS_TYPES_H 
-#define _SYS_UIO_H 
-#define _SYS_UN_H 
-#define _SYS_UTSNAME_H 
-#define _TAR_H 
-#define _TGMATH_H 
-#define _THREADS_H 
-#define _TIME_H 
-#define _UCHAR_H 
-#define _UNISTD_H 
-#define _UTIME_H 
-#define _VALUES_H 
-#define _VA_LIST 
-#define _WCHAR_H 
-#define _WCHAR_T 
-#define _WCTYPE_H 
-#define _WINT_T 
+#define _SEARCH_H
+#define _SEMAPHORE_H
+#define _SIZE_T
+#define _STDALIGN_H
+#define _STDBOOL_H
+#define _STDC_PREDEF_H
+#define _STDINT_H
+#define _STDIO_EXT_H
+#define _STDIO_H
+#define _STDLIB_H
+#define _STDNORETURN_H
+#define _STRINGS_H
+#define _STRING_H
+#define _STROPTS_H
+#define _SYSEXITS_H
+#define _SYS_EVENTFD_H
+#define _SYS_FILE_H
+#define _SYS_IOCTL_H
+#define _SYS_PARAM_H
+#define _SYS_RANDOM_H
+#define _SYS_REG_H
+#define _SYS_SELECT_H
+#define _SYS_SOCKET_H
+#define _SYS_STAT_H
+#define _SYS_SYSCALL_H
+#define _SYS_SYSINFO_H
+#define _SYS_TIMEB_H
+#define _SYS_TIMEX_H
+#define _SYS_TIME_H
+#define _SYS_TTYDEFAULTS_H
+#define _SYS_TYPES_H
+#define _SYS_UIO_H
+#define _SYS_UN_H
+#define _SYS_UTSNAME_H
+#define _TAR_H
+#define _TGMATH_H
+#define _THREADS_H
+#define _TIME_H
+#define _UCHAR_H
+#define _UNISTD_H
+#define _UTIME_H
+#define _VALUES_H
+#define _VA_LIST
+#define _WCHAR_H
+#define _WCHAR_T
+#define _WCTYPE_H
+#define _WINT_T
```

Comment:
Whats up with the whitespace changes here?

Reply:
Probably my IDE ripping out trailing whitespace? Should it not?

Reply:
These files are auto-generated, and should not be edited by hand.  If you want to remove the spaces (not sure its worth doing) then I would suggest doing is part of that auto-generation process.

Reply:
If you do want to do this we should make it a separate PR to keep this one focused.

(BTW I personally recommend against automatically stripping trailing white space on save, and instead binding a keyboard shortcut to it.. then you are in control of if/when you want to do the stripping, which makes it easier to work on lots of different projects, when not all of them are already free of trailing whitespace.  I use leader-w in vim for this).